tests/system/python/packages/test_multiple_assets.py

```diff
@@ -203,47 +214,48 @@ def test_multiple_assets_with_restart(self, remove_and_add_pkgs, reset_fledge, s
                 on endpoint GET /fledge/asset"""
 
         total_benchmark_services = 6
-        total_assets = PER_BENCHMARK_ASSET_COUNT * total_benchmark_services
+        num_assets_per_service = (num_assets//total_benchmark_services)
+        total_assets = num_assets_per_service * total_benchmark_services
 
         for count in range(total_benchmark_services):
             service_name = BENCHMARK_SOUTH_SVC_NAME + "{}".format(count + 1)
-            add_benchmark(fledge_url, service_name, count + 1)
+            add_benchmark(fledge_url, service_name, count + 1, num_assets_per_service)
             verify_service_added(fledge_url, service_name)
-
-        # Wait until total_assets are created
-        time.sleep(PER_BENCHMARK_ASSET_COUNT + 2 * wait_time)
+        
+        # Sleep for few seconds, So that some data can be ingested into the Fledge
+        time.sleep(wait_time * 3)
+        
         verify_ping(fledge_url, skip_verify_north_interface, wait_time, retries)
-        verify_asset(fledge_url, total_assets)
+        verify_asset(fledge_url, total_assets, num_assets//100, wait_time)
```

Comment:
what is num_assets//100 here and how it map to count arg?

Reply:
num_asset//100 gives the integer value, and 100 assets requires 60 second for creation.
So, the num_asset//100 gives number of iterations or count for which the loop defined in verify_asset function should execute, and that loop contains "wait_time * 6" == 60 seconds.